src/logic/small.lean

```diff
@@ -62,25 +62,21 @@ section
 open_locale classical
 
 theorem small_map {α : Type*} {β : Type*} [hβ : small.{w} β] (e : α ≃ β) : small.{w} α :=
-begin
-  tactic.unfreeze_local_instances,
-  rcases hβ with ⟨γ, ⟨f⟩⟩,
-  exact small.mk' (e.trans f)
-end
+small.mk' (e.trans (equiv_shrink β))
```

Comment:
I don't know if someone (@digama0 ?) cares about usage of `classical.choice` in the proofs in this file. The new proof uses this axiom, the old one doesn't.

Reply:
It's easy to avoid it:
```suggestion
let ⟨γ, ⟨f⟩⟩ := hβ.equiv_small in small.mk' (e.trans f)
```